scripts/maintenance/computational-clusters/autoscaled_monitor/core.py

```diff
@@ -776,3 +776,45 @@ async def trigger_cluster_termination(
 
 async def check_database_connection(state: AppState) -> None:
     await db.check_db_connection(state)
+
+
+async def terminate_dynamic_instances(
```

Comment:
Q: what does this imply? Will an instance be terminated if a service is running on top of it?